src/MLJGLMInterface.jl

```diff
@@ -300,19 +309,8 @@ function glm_formula(model, features::AbstractVector{Symbol})::FormulaTerm
     # Adding a zero term explicitly disables the intercept.
     # See the StatsModels.jl tests for more information.
     intercept_term = model.fit_intercept ? 1 : 0
-    form = GLM.Term(:y) ~ sum(GLM.term.(features)) + GLM.term(intercept_term)
-
-    # Workaround https://github.com/JuliaStats/StatsModels.jl/issues/289.
-    fixed_form = let
-        FormulaTerm = GLM.StatsModels.FormulaTerm
-        fixed_rhs = if form.rhs[1] isa GLM.StatsModels.ConstantTerm
-            (form.rhs[2:end]..., form.rhs[1])
-        else
-            form.rhs
-        end
-        FormulaTerm(form.lhs, fixed_rhs)
-    end
-
+    form = FormulaTerm(Term(:y), sum(term.(features)) + term(intercept_term))
```

Comment:
while in theory we may re-order terms as we see fit, I think at this point it woudl be pretty disruptive to introduce that logic at teh `+` or `sum` level, so this is like 99.9% safe IMO.